nncf/quantization/algorithms/min_max/onnx_backend.py

```diff
@@ -176,32 +179,36 @@ def _get_reduction_shape_and_use_abs_max(
         return tuple(reduction_shape), use_abs_max
 
     @staticmethod
-    def minmax_statistic_collector(nncf_graph: NNCFGraph,
-                                   target_point: ONNXTargetPoint,
-                                   quantizer_config: QuantizerConfig,
-                                   inplace: bool,
-                                   num_samples: int = None,
-                                   ) -> ONNXMinMaxStatisticCollector:
+    def get_statistic_collector(
+        range_estimator_params: RangeEstimatorParameters,
+        nncf_graph: NNCFGraph,
+        target_point: ONNXTargetPoint,
+        quantizer_config: QuantizerConfig,
+        inplace: bool,
+        num_samples: int = None) -> Union[ONNXMinMaxStatisticCollector, ONNXMeanMinMaxStatisticCollector]:
+
         reduction_shape, use_abs_max = \
             ONNXMinMaxAlgoBackend._get_reduction_shape_and_use_abs_max(nncf_graph,
                                                                        target_point,
                                                                        quantizer_config)
-        return ONNXMinMaxStatisticCollector(use_abs_max, reduction_shape, num_samples)
 
-    @staticmethod
-    def mean_minmax_statistic_collector(nncf_graph: NNCFGraph,
-                                        target_point: ONNXTargetPoint,
-                                        quantizer_config: QuantizerConfig,
-                                        use_per_sample_stats: bool,
-                                        inplace: bool,
-                                        num_samples: int = None,
-                                        ) -> ONNXMeanMinMaxStatisticCollector:
-        reduction_shape, use_abs_max = \
-            ONNXMinMaxAlgoBackend._get_reduction_shape_and_use_abs_max(nncf_graph, target_point,
-                                                                       quantizer_config)
-        return ONNXMeanMinMaxStatisticCollector(use_per_sample_stats,
-                                                use_abs_max, reduction_shape,
-                                                num_samples, window_size=None)
+        if (range_estimator_params.min.statistics_type == StatisticsType.MIN and
```

Comment:
Why don't we use here the same approach with `*_REDUCERS_MAP`?

Reply:
Because only OV backend implements reducers by now. All other backends are still using legacy tensor collectors